core/src/main/scala/scala/macros/core/Universe.scala

```diff
@@ -92,5 +98,18 @@ trait Universe {
   // Expansion
   // =========
   type Expansion
+  def enclosingPosition: Position
+  def enclosingOwner: Symbol
+}
+
+trait Input extends Any {
+  def path: Path
+}
 
+trait Position extends Any {
+  def start: Int
+  def end: Int
```

Comment:
Is there any known important usage of `start` and `end`? Otherwise, I think it's better to remove them for now if they are not widely and reliably used.

Reply:
I was gonna use them to implement https://github.com/lihaoyi/sourcecode/blob/420bea7941d3219e2f1200b14b11010843aea39c/sourcecode/shared/src/main/scala/sourcecode/SourceContext.scala#L130-L141 but then hit on #28. I can remove them for now until we have a test case using them.

Reply:
Also removed column since it's not used.